frontend/homework/16/src/redux/SearchSlice.ts

```diff
@@ -0,0 +1,22 @@
+import { PayloadAction, createSlice } from "@reduxjs/toolkit";
```

Comment:
its an overkill to use redux for storing a value. Use prop-drilling or context if you really want.